src/main/java/types/HostCpuPinning.java

```diff
@@ -0,0 +1,67 @@
+package types;
+
+import org.ovirt.api.metamodel.annotations.Link;
+import org.ovirt.api.metamodel.annotations.Type;
+
+/**
+ * Type representing a physical CPU of a host with
+ * the current pinning status.
+ *
+ * @author Lucia Jelinkova <ljelinko@redhat.com>
+ * @since 4.5
+ * @date 15 Feb 2022
+ * @status added
+ */
+@Type
+public interface HostCpuPinning extends Identified {
```

Comment:
Don't you wish to have the CpuPinningPolicy?

Reply:
I think it is not necessary as we link the pinned VMs and if the user needs to know the policy, it can be checked there.